pkg/types/validation/installconfig_test.go

```diff
@@ -2829,6 +2829,15 @@ func TestValidateTNF(t *testing.T) {
 			name:     "invalid_number_of_credentials_for_dual_replica",
 			expected: "controlPlane.fencing.credentials: Forbidden: there should be exactly two fencing credentials to support the two node cluster, instead 3 credentials were found",
 		},
+		{
+			config: installConfig().
+				MachinePoolArbiter(machinePool()).
+				MachinePoolCP(machinePool().Credential(c1(), c2())).
+				ArbiterReplicas(1).
+				CpReplicas(2).build(),
```

Comment:
nit: This looks like a valid setup for fencing credentials (i.e. see the case `"valid_two_credentials"`). We probably should set up an invalid case, for example, 3 credentials to check if the validation is actually skipped?

Reply:
Sure thing, let me add that now

Reply:
Added check

Reply:
Nice, thanks!